Update catalog.md (#7329)
braze-inc/braze-docs: _docs/_user_guide/personalization_and_dynamic_content/catalogs/catalog.md

You can also manually piece together catalogs Liquid logic. However, note that if you type in an ID that doesn't exist, Braze will still return an items array without objects. We recommend that you include error handling, such as checking the size of the array and using an `if` statement to account for an empty array case.

{% alert note %}

> Liquid can't be used inside catalogs. If you have Liquid personalization listed in a cell in your catalog, it won't render with the dynamic value; it will just display the actual Liquid.

Reviewer: ```suggestion
Liquid currently can't be used inside catalogs. If Liquid personalization is listed inside a cell in your catalog, the dynamic value won't render and only the actual Liquid will display.
```
